packages/fxa-settings/src/components/Settings/PageDeleteAccount/index.tsx

```diff
@@ -27,7 +26,7 @@ type FormData = {
 
 const checkboxLabels: Record<string, string> = {
   'delete-account-chk-box-1-v3':
```

Comment:
With the next push, I'll change this to `'delete-account-chk-box-1-v4'`. I had this locally and apparently didn't save.